arm/capture.c

```diff
@@ -760,13 +751,14 @@ static snd_pcm_sframes_t capture_wait(CaptureContext *restrict ctx)
 }
 
 
-void capture_period(
+void capture_capture_period(
```

Comment:
This method name is confusing. Can we choose something else? Perhaps `do_capture` here, and `get_capture_period` below?